workflow/snakemake_rules/core.smk

```diff
@@ -0,0 +1,295 @@
+'''
+This file contains rules that run the canonical augur pipeline on a set of sequences
+and generate a tree and a number of node_jsons
+
+input:
+ - builds/{build_name}/strains.txt
+ - builds/{build_name}/metadata.tsv
+ - builds/{build_name}/{segment}/sequences.fasta
+
+output:
+ - builds/{build_name}/{segment}/tree_raw.nwk
+ - builds/{build_name}/{segment}/tree.nwk
+ - builds/{build_name}/{segment}/branch-lengths.json
+ - builds/{build_name}/{segment}/aa-muts.json
+ - builds/{build_name}/{segment}/nt-muts.json
+ - builds/{build_name}/{segment}/traits.json
+'''
+
+localrules: clades, sanitize_trees
+
+build_dir = config.get("build_dir", "builds")
```

Comment:
Using this workflow more, I was surprised by this directory as the place to look for my output. The rest of our workflows use `results/` with the contents of that directory depending on the pathogen workflow (`zika` stores a single set of outputs top-level while `ncov` nests build-specific outputs in subdirectories). Can we switch back to that here?